src/index.ts

```diff
@@ -587,6 +587,8 @@ export class J5IO extends AbstractIO {
     } else if (Array.isArray(registerOrInBytes)) {
       register = undefined;
       value = registerOrInBytes;
+    } else if (typeof inBytes === 'number') {
```

Comment:
Assuming that my above interpretation of the signature is correct, I think what we want to do is change line 581 to read:

```typescript
if (typeof registerOrInBytes === 'number' && (Array.isArray(inBytes) || typeof inBytes === 'number')) {
  if (!Array.isArray(inBytes)) {
    inBytes = [ inBytes ]
  }
```

and then remove this section. I'm not 100%, but I think we need the first half of the signature check too for this to work.
```